exercises/book-store/book_store_test.rb

```diff
@@ -0,0 +1,92 @@
+require 'minitest/autorun'
+require_relative 'book_store'
+
+# Common test data version: 1.0.1 087ad69
```

Comment:
There's a new version of the canonical-data.
You probably need to update your problem-specifications repository and re-gererate the tests.